components/private/ShapeLibrary/src/MDCCurvedCornerTreatment.h

```diff
@@ -29,4 +29,16 @@
  */
 @property(nonatomic, assign) CGSize size;
 
+/**
+ Initializes an MDCCurvedCornerTreatment instance with a given corner size.
+ */
+- (nonnull instancetype)initWithSize:(CGSize)size NS_DESIGNATED_INITIALIZER;
+
+/**
+ Initializes an MDCCurvedCornerTreatment instance with a corner size of zero.
+ */
+- (nonnull instancetype)init;
+
+- (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_DESIGNATED_INITIALIZER;
```

Comment:
Break this out to a separate PR if possible please.

Reply:
https://github.com/material-components/material-components-ios/pull/3378